spec/features/user_signs_up_spec.rb

```diff
@@ -417,26 +408,25 @@
     end
 
     scenario 'required fields blank' do
-      complete_signup_profile_screen(
-        role: :instructor,
-        first_name: "",
-        last_name: "",
-        phone_number: "",
-        school: "",
-        url: "",
-        num_students: "",
-        using_openstax: "",
-        newsletter: true,
-        agree: true
-      )
-
-      [:first_name, :last_name, :phone_number, :school, :url, :using_openstax].each do |var|
-        expect(page).to have_content(error_msg SignupProfileInstructor, var, :blank)
-      end
-      expect(page).to have_content(error_msg SignupProfileInstructor, :num_students, :not_a_number)
-      expect(page).to have_content(error_msg SignupProfileInstructor, :subjects, :blank_selection)
-
-      screenshot!
+      # TODO Cannot submit blank fields in new form. Remove this?
+      # complete_signup_profile_screen(
+      #   role: :instructor,
+      #   first_name: "",
+      #   last_name: "",
+      #   phone_number: "",
+      #   school: "",
+      #   url: "",
+      #   using_openstax: "",
+      #   agree: true
+      # )
+      #
+      # [:first_name, :last_name, :phone_number, :school, :url, :using_openstax].each do |var|
+      #   expect(page).to have_content(error_msg SignupProfileInstructor, var, :blank)
+      # end
+      # expect(page).to have_content(error_msg SignupProfileInstructor, :num_students, :not_a_number)
+      # expect(page).to have_content(error_msg SignupProfileInstructor, :subjects, :blank_selection)
+      #
+      # screenshot!
     end
```

Comment:
we should still test that errors are shown when fields are left blank

Reply:
There's no active Next button when required fields are blank, so the attempt to submit (in complete_signup_profile_screen) fails.

Reply:
What I have done is tested that you can't click Next, or that Next does not change pages.